File: AspNetMvcUnleashed/0672329980_source/Chapter07/VB/MvcApplication1/MvcApplication1/Controllers/EmployeeController.vb
Public Class EmployeeController
    Inherits System.Web.Mvc.Controller

    ' GET: /Employee/Create
    Function Create() As ActionResult
        Return View()
    End Function

    ' POST: /Employee/Create
    <AcceptVerbs(HttpVerbs.Post)> _
    Function Create(ByVal employeeToCreate As Employee) As ActionResult
        If Not ModelState.IsValid Then
            Return View()
        End If

        ' Add employee to database
        Return RedirectToAction("Index")
    End Function


End Class

File: AspNetMvcUnleashed/0672329980_source/Chapter07/CS/MvcApplication1/MvcApplication1/Controllers/EmployeeController.cs
using System.Web.Mvc;
using MvcApplication1.Models;

namespace MvcApplication1.Controllers
{
    public class EmployeeController : Controller
    {

        //
        // GET: /Employee/Create

        public ActionResult Create()
        {
            return View();
        } 

        //
        // POST: /Employee/Create

        [AcceptVerbs(HttpVerbs.Post)]
        public ActionResult Create(Employee employeeToCreate)
        {
            if (!ModelState.IsValid)
                return View();

            // Add employee to database
            return RedirectToAction("Index");
        }

    }
}
